webpack/webpack.parts.js

```diff
@@ -170,6 +172,24 @@ exports.externals = () => ({
   },
 });
 
+exports.preloadFonts = () => ({
+  plugins: [
+    new HtmlWebpackPlugin({
+      filename: 'preload.html',
+      templateContent: '{{{preloadLinks}}}',
+      inject: false,
+    }),
+    new FontPreloadPlugin({
+      index: 'preload.html',
+      extensions: ['woff2'],
```

Comment:
Why only `woff2` I know `eot` is only for IE and we don't support it but `ttf` is supported by all browser.